python/setup.py

```diff
@@ -40,6 +40,12 @@
 # Check if we're running 64-bit Python
 is_64_bit = sys.maxsize > 2**32
 
+is_emscripten = (
+    sysconfig.get_config_var("SOABI")
+    and sysconfig.get_config_var("SOABI").find("emscripten") != -1
+)
```

Comment:
Yes, because in cross-compiling situations it appears `sys.platform` contains the build platform (i.e. linux / windows or whatever), whereas `sysconfig` contains the target platform (i.e. emscripten). So in setup.py, which is running on a host platform, you have to look as `sysconfig`, whereas in code that will actually run on emscripten, you can use sys.platform.

Reply:
OK, thanks for the explanation. Can you add something like that as a comment here?